libs/ballot-encoder/src/index.test.ts

```diff
@@ -84,13 +86,15 @@ test('can detect our legacy formats', () => {
 });
 
 test('encodes & decodes with Uint8Array as the standard encoding interface', () => {
-  const { election, electionHash } = electionDefinition;
-  const ballotStyle = election.ballotStyles[0]!;
-  const precinct = election.precincts[0]!;
+  const { electionHash } = electionDefinition;
```

Comment:
Did you ever look into providing the hash from the ADT? We'd have to build the ADT off of `electionData` rather than the object, but it could take out a step.

And any reason why you'd can't define the ADT election at the top level apart from any particular test, and reuse, if you don't want to repeat this conversion?